js/libs/keycloak-js/src/keycloak.js

```diff
@@ -450,7 +450,7 @@ function Keycloak (config) {
             url += '&prompt=' + encodeURIComponent(options.prompt);
         }
 
-        if (options && options.maxAge) {
+        if (options && (options.maxAge || options.maxAge === 0)) {
```

Comment:
Rather than to assert the value being `0` we can opt to check if the value is a number instead.
```suggestion
        if (options && typeof options.maxAge === 'number') {
```

Reply:
Thanks, I have updated the code to be like this.

Infact I had thought of doing this but, just did not want to loose any other thing we may be doing with this option, in case we were passing any non-numeric values.
From what I know there should never be a non numeric value going into max_age, but was in doubt due to the use of unnecessary encodeURIComponent done on the number. But I understand now that is needed for avoiding injection vulnerabilities.

Note that I added a !isNaN check to avoid sending invalid number for `NaN`, as `typeof NaN` evaluates to true.
Let me know if this is okay or I should revert to not having !isNaN()

Reply:
Indeed, this could possibly be passed as a string, but the type we expect is a number (also in the TypeScript types), so it's fine to run on this assumption.